base/operators.jl

```diff
@@ -944,6 +944,9 @@ Function composition also works in prefix form: `∘(f, g)` is the same as `f 
 The prefix form supports composition of multiple functions: `∘(f, g, h) = f ∘ g ∘ h`
 and splatting `∘(fs...)` for composing an iterable collection of functions.
 
+In the splatting of a composed function, say `fs[...]`, functions are called in
+descending order, i.e. `func in fs[end:-1:begin]`, till the last function is called.
```

Comment:
Why is splatting relevant? Just above the example has `f ∘ g ∘ h`, and it might be enough just to clarify that `h` is called first, i.e. this is `f(g(h(x)))`. 

Although the first line says "`(f ∘ g)(args...; kwargs...)` means `f(g(args...; kwargs...))`". Does this not already imply that `h` is called first?